src/Microsoft.AspNetCore.Mvc.Core/Microsoft.AspNetCore.Mvc.Core.csproj

```diff
@@ -11,6 +11,7 @@ Microsoft.AspNetCore.Mvc.FromFormAttribute
 Microsoft.AspNetCore.Mvc.RequireHttpsAttribute
 Microsoft.AspNetCore.Mvc.RouteAttribute</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Microsoft.AspNetCore.Mvc</RootNamespace>
```

Comment:
How is this change (which brings in a lot of additional files) related to the IVT one? In addition, the ambiguous `Resources` references you had to clean up get quite hideous e.g. `AspNetCore.Mvc.RazorPages.Resources.ArgumentCannotBeNullOrEmpty`.